app/src/main/kotlin/com/wire/android/ui/common/imagepreview/BulletHoleImagePreview.kt

```diff
@@ -67,7 +84,7 @@ fun BulletHoleImagePreview(
                 }
         ) {
             Image(
-                painter = rememberAsyncImagePainter(imageUri.toBitmap(LocalContext.current)),
+                painter = rememberAsyncImagePainter(loadBiMap(imageUri).value),
```

Comment:
question: can't coil handle URIs, like `rememberAsyncImagePainter(imageUri)`? Do we need to convert uri to bitmap ourselves or could we just let coil do it for us inside this async image painter?

Reply:
Looks like `rememberAsyncImagePainter` wraps data by default into `ImageRequest` and it [supports URI](https://github.com/coil-kt/coil/blob/dc5707ba29ccd42575f0b2d161f70bfe1cb9ce69/coil-base/src/main/java/coil/request/ImageRequest.kt#L449), so we can just pass `imageUri`, right?

Reply:
Yes it works if we just pass the uri, but internally Coil loads the image on Main 🤷‍♂️
<img width="1499" alt="Screenshot 2023-09-20 at 14 55 45" src="https://github.com/wireapp/wire-android-reloaded/assets/18124919/3aa08920-c0f2-4d32-8e71-14c1a8a47ae2">